mesh_led_control_example.cpp

```diff
@@ -103,6 +110,7 @@ static void receive() {
         int length = my_socket->recvfrom(&source_addr, receive_buffer, sizeof(receive_buffer) - 1);
         if (length > 0) {
             tr_debug("Packet from %s\n", source_addr.get_ip_address());
+            messageTimeout.attach(&messageTimeoutCallback, MESSAGE_WAIT_TIMEOUT);
```

Comment:
random distribution is needed so that not all devices send the message simultaneously